sroze/openwhisk-bundle/0.2/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "index.php": "index.php"
+    },
+    "gitignore": [
+        "/.serverless",
+        "/node_modules"
```

Comment:
This one is not related to the bundle, but more generic. Is it something we want by default in the `symfony/framework-bundle` recipe?

Reply:
Actually.. none of these are related to openwhisk directly either 🤔 I'll remove them from this PR. 

`/node_modules/` is already in the [`webpack-encore-pack` manifest](https://github.com/symfony/recipes/blob/master/symfony/webpack-encore-pack/1.0/manifest.json#L9) so I guess it's alright for us.